.storybook/themes/commerce.scss

```diff
@@ -680,6 +680,46 @@
 						),
 					),
 				),
+				right-rail: (
+					display: grid,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+				),
+				right-rail-full-width-1: (
```

Comment:
Why is the full-width section set to display none?

Reply:
Because its not used at least in the case of the product description page

Reply:
@matthewroach I have updated the code base to reflect our discussion in huddle on Monday 7/18